src/shellbar.scss

```diff
@@ -41,23 +52,23 @@ $block: #{$fd-namespace}-shellbar;
   justify-content: space-between;
 
   @include fd-screen(s) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-small;
+    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr;
   }
 
   @include fd-screen(m) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-medium;
+    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr;
   }
 
   @include fd-screen(l) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-large;
+    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr;
   }
 
   @include fd-screen(xl) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-large;
+    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr;
   }
 
   @include fd-screen(xxl) {
-    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr-large;
+    padding: $fd-shellbar-outer-spacing-tb $fd-shellbar-outer-spacing-lr;
   }
```

Comment:
looks like the padding doesn't change, you can remove this and just add `padding: 0 1rem;`